### PR TITLE
Add key and verifyChain connection options

### PR DESCRIPTION
**Pull request by mitch-roblox**
This adds options for `key` and `verifyChain` to the per-server settings.

### CODE REVIEW

**Comment by mitch-roblox** [comment]
Hi, just checking to see if this can be merged?

**Comment by martinisoft** [comment]
@mitch-roblox right now you have two conflicts so you'll have to rebase against master for us to consider merging this.

**Comment by mitch-roblox** [comment]
Hi, I rebased this against master and now it's passing the tests. Please let me know if there is anything else you need from me. Thanks!

**Comment by mitch-roblox** [comment]
@martinisoft Hi, just checking in on this. Thanks!

File: templates/default/stunnel.conf.erb [hunk 1]
```diff
@@ -99,4 +99,7 @@ TIMEOUTclose = <%= opts['timeout_close'] ? 1 : 0 %>
 <% unless opts['client'].nil? -%>
 client = <%= opts['client'] ? "yes" : "no" %>
 <% end -%>
+<% unless opts['verify_chain'].nil? %>
```
Comment (martinisoft) [inline templates/default/stunnel.conf.erb]: Little nitpick, but I'd like you to add the whitespace chomp at the end of the tag like the other unless blocks for consistency. At the end change `%>` to `-%>` on this line and the end line as well.

**Comment by mitch-roblox** [comment]
@martinisoft Ah, good catch, thank you! I've pushed up the requested fix.

**Comment by mitch-roblox** [comment]
@martinisoft Hello, just checking in on this. Thank you!

**Comment by martinisoft** [comment]
I'll work on merging and shipping this later today. Have a bit of a backlog to work through first. Thank you again for your contribution. ❤️

**Comment by mitch-roblox** [comment]
Hello, I hate to be a bother, but just checking in on this. Thank you!

**Comment by mitch-roblox** [comment]
@martinisoft Hi, just checking in on this. I would greatly appreciate it if it could be merged soon. Thank you!

**Comment by martinisoft** [comment]
Thanks for checking in @mitch-roblox I'll get to a release shortly. On conference wifi so we'll see how well this works.

**Comment by martinisoft** [comment]
All merged and shipped. It should be available on the supermarket shortly. 👍

**Pull request merged**